store/rootmulti/store.go

```diff
@@ -439,7 +439,11 @@ func (rs *Store) Commit() types.CommitID {
 	defer rs.flushMetadata(rs.db, version, rs.lastCommitInfo)
 
 	// remove remnants of removed stores
-	for sk := range rs.removalMap {
+	keys := make([]types.StoreKey, 0, len(rs.removalMap))
+	for key := range rs.removalMap {
+		keys = append(keys, key)
+	}
+	for _, sk := range keys {
```

Comment:
This code isn't sorting the keys, so AFAICT doesn't produce behavior any different than ranging over the map directly. Am I missing something?